### DISCUSSION TITLE
Is it possible to use .md extensions for files rather than .qmd so that the files are compatible with other markdown platforms too?

### DISCUSSION THREAD

**Discussion by asifshakir** [quarto-dev/quarto-cli#10046] [Q&A] [answered]
### Description

Is it possible to use .md extensions for files rather than .qmd so that the files are compatible with other markdown platforms too?

**Comment by cderv** [comment, marked as answer]
Quarto can be used with `.md` files, but in this case no computation engine can be used. 

Using a computation engine requires to use `.qmd` as extension. A Quarto project can mix .md and .qmd file without problem. 

See also this also this related discussion on the reason .qmd is used when intermediary output of computation are expected
- https://github.com/quarto-dev/quarto-cli/discussions/3715

> compatible with other markdown platforms too?

What Markdown platform are you referring to ? What compatibility are you looking for or expecting ? 

Thanks !

**Reply by asifshakir** [reply]
Ah Ok, that sounds very cool.

I am evaluating quarto for a technical documentation platform and do not need the computational engine component.

I am using the same set of markdown files for hugo, and direct pandoc commands.

**Reply by mcanouil** [reply]
FYI, Quarto supports Hugo.
